README.md

```diff
@@ -1,3 +1,6 @@
+* [![Build Status](https://travis-ci.com/rabbitmq/cuttlefish.svg?branch=fix-tests)](https://travis-ci.com/Kyorai/cuttlefish)
```

Comment:
The first URL stills points to the rabbitmq organization on Travis CI, but it reflects the status of our fork, not upstream.